shlr/ptrace-wrap/src/ptrace_wrap.c

```diff
@@ -78,6 +79,11 @@ static void wrap_func(ptrace_wrap_instance *inst) {
 }
 
 static void *th_run(ptrace_wrap_instance *inst) {
+	sigset_t set;
+	sigemptyset (&set);
+	sigaddset (&set, SIGINT);
```

Comment:
uhm not sure if we want a ^C handler here

Reply:
This is no longer necessary and will be removed.